web/src/yarn.lock

```diff
@@ -1,11981 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
we don't want this deleted, you can regenerate with `yarn install` from within `web/src`

Reply:
Might be a yarn version discrepancy too:

```bash
$ yarn --version
1.17.3
```